app/controllers/patients_controller.rb

```diff
@@ -314,6 +314,19 @@ def update_status
     patient = current_user.get_patient(params.permit(:id)[:id])
     redirect_to(root_url) && return if patient.nil?
 
+    # update LDE for patient and group members only in the exposure workflow with continuous exposure on (separate from updating monitoring status)
+    if params.permit(:apply_to_group_cm_exp_only)[:apply_to_group_cm_exp_only]
+      ([patient] + (current_user.get_patient(patient.responder_id)&.dependents&.where(continuous_exposure: true, isolation: false) || [])).uniq.each do |member|
```

Comment:
most of the logic in this block of code is similar to the logic for `apply_to_group_cm_only` and it might be worth refactoring it out into a separate method with the patients (including household members) and lde_date as params or something

Reply:
done